### PR TITLE
Updated comments and fixed exceptions

### PR DESCRIPTION
**Pull request by KostinP**

### CODE REVIEW

**Review by lalagvana — APPROVED** [review]

File: src/AuthenticationService.Business/UserLoginCommand.cs [hunk 1]
```diff
@@ -56,7 +67,7 @@ private async Task<IUserCredentialsResponse> GetUserCredentials(string userEmail
 
             if (!brokerResponse.Message.IsSuccess)
             {
-                throw new Exception(String.Join(", ", brokerResponse.Message.Errors));
+                throw new ForbiddenException(String.Join(", ", brokerResponse.Message.Errors));
             }
```
Comment (djino97) [inline src/AuthenticationService.Business/UserLoginCommand.cs]: Use StringBuilder

Comment (KostinP) [reply src/AuthenticationService.Business/UserLoginCommand.cs]: Fixed.

File: src/AuthenticationService.Business/AuthenticationService.Business.csproj [hunk 2]
```diff
@@ -7,8 +7,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="FluentValidation" Version="9.0.1" />
-    <PackageReference Include="LT.DigitalOffice.Kernel" Version="1.0.2" />
+    <PackageReference Include="FluentValidation" Version="9.2.2" />
```
Comment (djino97) [inline src/AuthenticationService.Business/AuthenticationService.Business.csproj]: remove

File: src/AuthenticationService/AuthenticationService.csproj [hunk 3]
```diff
@@ -7,14 +7,14 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="FluentValidation" Version="9.0.1" />
+    <PackageReference Include="FluentValidation" Version="9.2.2" />
```
Comment (djino97) [inline src/AuthenticationService/AuthenticationService.csproj]: remove

File: src/AuthenticationService.Broker/AuthenticationService.Broker.csproj [hunk 4]
```diff
@@ -7,7 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="LT.DigitalOffice.Kernel" Version="1.0.2" />
+    <PackageReference Include="LT.DigitalOffice.Kernel" Version="1.0.9" />
     <PackageReference Include="MassTransit" Version="7.0.4" />
```
Comment (djino97) [inline src/AuthenticationService.Broker/AuthenticationService.Broker.csproj]: remove MassTransit, it have in kernel, and remove it in all project where kernel having

**Review by DarkFelas — APPROVED** [review]

File: src/AuthenticationService.Business/AuthenticationService.Business.csproj (hunk 2 above)
Comment (KostinP) [reply src/AuthenticationService.Business/AuthenticationService.Business.csproj]: Fixed.

File: src/AuthenticationService/AuthenticationService.csproj (hunk 3 above)
Comment (KostinP) [reply src/AuthenticationService/AuthenticationService.csproj]: Fixed.

File: src/AuthenticationService.Broker/AuthenticationService.Broker.csproj (hunk 4 above)
Comment (KostinP) [reply src/AuthenticationService.Broker/AuthenticationService.Broker.csproj]: Fixed.

**Pull request closed without merging**